resources/fixtures/Generic/Generic-Generic-Smoke.qxf

```diff
@@ -1,34 +1,36 @@
-<?xml version='1.0' encoding='UTF-8'?>
+<?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE FixtureDefinition>
 <FixtureDefinition xmlns="http://www.qlcplus.org/FixtureDefinition">
  <Creator>
   <Name>Q Light Controller Plus</Name>
-  <Version>4.10.6 GIT</Version>
+  <Version>4.12.4</Version>
   <Author>Thierry Rodolfo</Author>
  </Creator>
  <Manufacturer>Generic</Manufacturer>
  <Model>Generic Smoke</Model>
  <Type>Smoke</Type>
- <Channel Name="Pump">
-  <Group Byte="0">Intensity</Group>
-  <Capability Min="0" Max="255">Min &lt; Max</Capability>
- </Channel>
+ <Channel Name="Pump" Preset="IntensityMasterDimmer"/>
  <Channel Name="Fan">
-  <Group Byte="0">Intensity</Group>
-  <Capability Min="0" Max="255">Min &lt; Max</Capability>
+  <Group Byte="0">Speed</Group>
+  <Capability Min="0" Max="255" Preset="SlowToFast">Fan Speed, slow - fast</Capability>
  </Channel>
- <Mode Name="Normal">
+ <Channel Name="Fan 2" Preset="IntensityDimmer"/>
+ <Mode Name="1 Channel">
```

Comment:
Renaming modes means breaking existing projects. Please don't do that unless it's a definition mistake.